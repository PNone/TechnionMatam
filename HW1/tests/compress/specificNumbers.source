baraa adi 1498 20:21
baraa adi 4832 20:20
baraa adi 858172 20:20
baraa adi 50 20:20
baraa adi 492 20:20
baraa liav 341 20:19
baraa adi 3921 20:19