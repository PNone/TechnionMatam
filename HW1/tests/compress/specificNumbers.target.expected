BlockChain Info:
1.
Sender Name: baraa
Receiver Name: adi
Transaction Value: 865044
Transaction timestamp: 20:21
2.
Sender Name: baraa
Receiver Name: liav
Transaction Value: 341
Transaction timestamp: 20:19
3.
Sender Name: baraa
Receiver Name: adi
Transaction Value: 3921
Transaction timestamp: 20:19
